setup.sh

```diff
@@ -97,72 +97,104 @@ function _inspect
 
 function _usage
 {
-  echo "${SCRIPT:-${0}} Bootstrapping Script
-
-Usage: ${0} [-i IMAGE_NAME] [-c CONTAINER_NAME] <subcommand> <subcommand> [args]
-
-OPTIONS:
-
-  -i IMAGE_NAME     The name of the docker-mailserver image
-                    The default value is
-                    'docker.io/mailserver/docker-mailserver:latest'
-
-  -c CONTAINER_NAME The name of the running container.
-
-  -p PATH           Config folder path (default: ${CDIR}/config)
-
-  -h                Show this help dialogue
-
-  -z                Allow container access to the bind mount content
-                    that is shared among multiple containers
-                    on a SELinux-enabled host.
-
-  -Z                Allow container access to the bind mount content
-                    that is private and unshared with other containers
-                    on a SELinux-enabled host.
-
-SUBCOMMANDS:
-
-  email:
-
-    ${0} email add <email> [<password>]
-    ${0} email update <email> [<password>]
-    ${0} email del [OPTIONS] <email>
-    ${0} email restrict <add|del|list> <send|receive> [<email>]
-    ${0} email list
-
-  alias:
-
-    ${0} alias add <email> <recipient>
-    ${0} alias del <email> <recipient>
-    ${0} alias list
-
-  quota:
-
-    ${0} quota set <email> [<quota>]
-    ${0} quota del <email>
-
-  config:
-
-    ${0} config dkim <keysize> (default: 4096) <domain> (optional - for LDAP systems)
-    ${0} config ssl <fqdn>
-
-  relay:
-
-    ${0} relay add-domain <domain> <host> [<port>]
-    ${0} relay add-auth <domain> <username> [<password>]
-    ${0} relay exclude-domain <domain>
-
-  debug:
-
-    ${0} debug fetchmail
-    ${0} debug fail2ban [<unban> <ip-address>]
-    ${0} debug show-mail-logs
-    ${0} debug inspect
-    ${0} debug login <commands>
-
-  help: Show this help dialogue
-
+  echo -e "\e[35mSETUP\e[31m(\e[93m1\e[31m)
+
+\e[38;5;214mNAME\e[39m
+    ${SCRIPT:-${0}} - docker-mailserver administration script
+
+\e[38;5;214mSYNOPSIS\e[39m
+    ./${SCRIPT:-${0}} [ OPTIONS\e[31m...\e[39m ] OBJECT COMMAND [ help \e[31m|\e[39m ARGUMENTS\e[31m...\e[39m ]
```

Comment:
According to your screenshot it is:
`./setup.sh [OPTIONS] OBJECT COMMAND [help | ARGUMENTS]` which I find misleading as ongoing `OBJECTS` are elements of `COMMANDS` so this:
`./setup.sh [OPTIONS] COMMAND [help | ARGUMENTS]` seems more valid to me.

Reply:
I have altered it to now use an easier to understand schema. I have added a `SUBCOMMAND` after the `COMMAND` definition in this line:

``` BASH
COMMAND := { email | alias | quota | config | relay | debug } SUBCOMMAND
```

which makes all clear I think.